openupgrade_scripts/apriori.py

```diff
@@ -49,6 +49,8 @@
     # OCA/project
     "project_mail_chatter": "project",
     "project_task_dependency": "project",
+    # OCA/timesheet
+    "sale_timesheet_order_line_sync": "sale_project",
```

Comment:
```suggestion
    "sale_timesheet_order_line_sync": "sale_timesheet",
```

Reply:
Not really. The one that fits more is this one.

Reply:
Firstly, `sale_timesheet_order_line_sync` depends on `sale_timesheet`. And secondly, it's code is simple:

![Selection_713](https://user-images.githubusercontent.com/25005517/211779274-5a2aac6d-fa48-40c9-8597-8d0911787117.png)

and by looking similar things in the code of odoo 🔎:
- sale_project: nothing similar
- sale_project_account: nothing similar
- sale_timesheet:

![Selection_714](https://user-images.githubusercontent.com/25005517/211778959-a22a993e-6c0e-4252-954d-8872879a3ef1.png)
![Selection_716](https://user-images.githubusercontent.com/25005517/211779769-a0bf656e-216a-4cfc-a27f-a8e610bc7a8a.png)
![Selection_715](https://user-images.githubusercontent.com/25005517/211778995-bfd327b5-b43d-4ec8-98e7-e675b7972d48.png)

Oh, there are several similarities in the code of sale_timesheet, match?  🎉

BTW, If you think I am wrong, I think you should give your arguments about "fitting" 🤔

Reply:
OK, it doesn't matter anyway. Put again you request changes.